### PR TITLE
Added backers and sponsors from OpenCollective

### PR DESCRIPTION
**Pull request by piamancini**
Now backers & sponsors will appear automatically on your readme. added badges on top and buttons linking to your public page.

### CODE REVIEW

**Comment by matthewmueller** [comment]
@jugglinmike @davidchambers @fb55 planning to try out open collective to finance some of the support, possible refactoring and maybe a website (!)

Not sure if this is possible yet, but I'm planning on distributing any funds according to `git-summary` contributions.

Let me know if you have any thoughts or objections to this experiment.

**Comment by fb55** [comment]
Sounds good in general, although funds should be used IMHO for future development.

**Comment by davidchambers** [comment]
I'm interested in funding models for open-source projects, though I have no experience which such things. I agree with Felix that funds should be put towards future development.

While it may be apparent from my lack of recent contributions, this is a good time to state that I have my hands full with Sanctuary and related projects and don't intend to make further contributions to cheerio. I'm still happy to chime in when summoned. :)

File: Readme.md [hunk 1]
```diff
@@ -1001,6 +1001,74 @@ The core API is the best of its class and despite dealing with all the browser i
...
+<a href="https://opencollective.com/cheerio/backer/23/website" target="_blank"><img src="https://opencollective.com/cheerio/backer/23/avatar.svg"></a>
+<a href="https://opencollective.com/cheerio/backer/24/website" target="_blank"><img src="https://opencollective.com/cheerio/backer/24/avatar.svg"></a>
+<a href="https://opencollective.com/cheerio/backer/25/website" target="_blank"><img src="https://opencollective.com/cheerio/backer/25/avatar.svg"></a>
+<a href="https://opencollective.com/cheerio/backer/26/website" target="_blank"><img src="https://opencollective.com/cheerio/backer/26/avatar.svg"></a>
+<a href="https://opencollective.com/cheerio/backer/27/website" target="_blank"><img src="https://opencollective.com/cheerio/backer/27/avatar.svg"></a>
+<a href="https://opencollective.com/cheerio/backer/28/website" target="_blank"><img src="https://opencollective.com/cheerio/backer/28/avatar.svg"></a>
+<a href="https://opencollective.com/cheerio/backer/29/website" target="_blank"><img src="https://opencollective.com/cheerio/backer/29/avatar.svg"></a>
```
Comment (davidchambers) [inline Readme.md]: What is all this?

Comment (matthewmueller) [inline Readme.md]: they're placeholder images for sponsors :-) see: https://opencollective.com/apex for an example.

**Comment by matthewmueller** [comment]
sounds like everyone is onboard to try this. merged!

**Pull request merged**